index.js

```
@@ -27,7 +31,18 @@ const redisConnection = new IORedis({
 
 const queue = new Queue(queueName, { connection: redisConnection });
 
-app.post('/job', async (req, res) => {
+const authenticateToken = (req, res, next) => {
+  const authHeader = req.headers.authorization;
+  if (authHeader && authHeader.startsWith('Bearer ')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)